CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Developer Setup
+
+Ensure you have `kubectl`, `kind`, and some kind of `docker` command installed.
```

Comment:
Let's not forget about having `go` command in the PATH as well; kind of obvious when cloning the repo, but doesn't hurt mentioning 😸